satellite/peer.go

```diff
@@ -363,30 +374,93 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 		peer.Accounting.Rollup = rollup.New(peer.Log.Named("rollup"), peer.DB.Accounting(), config.Rollup.Interval)
 	}
 
+	{ // setup mailservice
+		log.Debug("Setting up mail service")
+		// TODO(yar): test multiple satellites using same OAUTH credentials
```

Comment:
should be easy to simulate with storj-sim

Reply:
Haven't tested it yet